File: Benchmarking_Languages_Project/Julia_Tests/Test14.jl
tic()
for i in 1:15
T14 = zeros(Int64,500,500)
 for j in 1:500
  for k in 1:500
   T14[k,j] = abs(j-k) + 1
   end
  end
end
timing = toc()
println (timing/15)

File: Benchmarking_Languages_Project/Python_Tests/Test14.py
from scipy import *
from numpy import *
from time import *

timing = zeros(15)
for i in range(15):
	time1 = time()
	T14 = zeros((500,500))
 	for j in range(500):
  		for k in range(500):
   			T14[k,j] = abs(j-k) + 1
	timing[i]=time()-time1
print mean(timing)
